test/unittests/libethcore/difficulty.cpp

```diff
@@ -214,7 +214,7 @@ BOOST_AUTO_TEST_CASE(difficultyTestsCustomMainNetwork)
 
 	if (dev::test::Options::get().filltests)
 	{
-		u256 homesteadBlockNumber = 3500000;
+		u256 homesteadBlockNumber = 4300000;
```

Comment:
Why?

Reply:
ok i haven't mention that. but this blocknumber is byzantine transition number

Reply:
it used to be the test cases around homestead number. now this is around byzantine

Reply:
Please change the name of the variable.